amcl/test/set_initial_pose.xml

```diff
@@ -32,19 +34,21 @@ setting pose: 30.329 34.644 3.142
       <param name="laser_lambda_short" value="0.1"/>
       <param name="laser_lambda_short" value="0.1"/>
       <param name="laser_model_type" value="likelihood_field"/>
-      <!-- <param name="laser_model_type" value="beam"/> -->
       <param name="laser_likelihood_max_dist" value="2.0"/>
+
       <param name="update_min_d" value="0.2"/>
       <param name="update_min_a" value="0.5"/>
       <param name="odom_frame_id" value="odom"/>
       <param name="resample_interval" value="1"/>
       <param name="transform_tolerance" value="0.1"/>
       <param name="recovery_alpha_slow" value="0.0"/>
       <param name="recovery_alpha_fast" value="0.0"/>
-      <!--param name="initial_pose_x" value="47.443"/>
+      <!--
+      <param name="initial_pose_x" value="47.443"/>
       <param name="initial_pose_y" value="21.421"/>
-      <param name="initial_pose_a" value="-1.003"/-->
+      <param name="initial_pose_a" value="-1.003"/>
```

Comment:
If you update the values below I think you should also update these values

Reply:
Sounds good

Reply:
No these shouldn't change, these aren't related. It is explainable why these are similar.